src/styles/ui-kit-buttons.module.scss

```diff
@@ -0,0 +1,71 @@
+@import '@styles/colors.scss';
+
+.container {
+    font-family: Wix Madefor Text;
+    width: 284px;
+    padding: 16px;
+}
+
+.uikit {
+    font-size: 10px;
+    font-weight: 700;
+    line-height: 12.6px;
+    letter-spacing: 0.1em;
+}
+
+.coreComponents {
+    font-size: 10px;
+    letter-spacing: 0.1em;
+}
+.active {
+    color: var(--black);
+    font-weight: 600;
+}
+
+.hrSolid {
+    height: 1px;
+    width: 250px;
+    background-color: #000;
+}
+
+.sectionTitle {
+    font-size: 12px;
+    font-weight: 800;
+    margin-top: 10px;
+}
+
+.hrLight {
+    height: 1px;
+    width: 250px;
+    background-color: #000;
+    opacity: 0.2;
+    margin-top: 40px;
+}
+
+.sectionHeader {
+    font-size: 11px;
+    font-weight: 800;
+    line-height: 12px;
+    letter-spacing: 0.12em;
+    color: #625d5d;
+    margin: 20px 0px;
+}
```

Comment:
`letter-spacing: 0.12em;` is really annoying but doesn't convert nicely to `px` so... whatever lets keep it.